src/net/sourceforge/kolmafia/textui/javascript/ValueConverter.java

```diff
@@ -217,6 +218,8 @@ else if (object instanceof Boolean) {
       return convertNativeArray((NativeArray) object, typeHint);
     } else if (object instanceof Value) {
       return (Value) object;
+    } else if (Undefined.isUndefined(object)) {
+      return null;
```

Comment:
This is a bit weird because `null` and `undefined` are different in JS. What's the benefit here?

Reply:
Well, at least it does not get returned as a string. How would you want to handle it?

Reply:
If anything, I would prefer to throw an exception. Neither Java nor Ash knows the concept of undefined in addition to null, so I'm not sure we can gain anything here.

Reply:
Maybe the issue is that we can coerce it to null operationally but we need to report it as undefined in the cli?

Reply:
I'll figure something out later this week.

Reply:
I think this will work: https://github.com/kolmafia/kolmafia/pull/2291